src/app/components/RoundActionButton/RoundActionButton.tsx

```diff
@@ -2,41 +2,58 @@ import React from 'react';
 import type { ReactNode } from 'react';
 import styled from 'styled-components';
 
-type DeleteButtonProps = {
+type RoundActionButtonProps = {
   handleDelete: () => void;
   children: ReactNode;
+  customStyles?: {
+    sizeButton?: string;
+    topPosition?: string;
+    rightPosition?: string;
+  };
 };
 
-export default function DeleteButton({
+export default function RoundActionButton({
   handleDelete,
```

Comment:
I'd rename the button because now it's a more generic button that doesn't only delete.

```suggestion
  handleClick,
```